cli/command/container/stats.go

```diff
@@ -31,7 +31,7 @@ func NewStatsCommand(dockerCli *command.DockerCli) *cobra.Command {
 	var opts statsOptions
 
 	cmd := &cobra.Command{
-		Use:   "stats [OPTIONS] [CONTAINER...]",
+		Use:   "stats [OPTIONS] CONTAINER [CONTAINER...]",
 		Short: "Display a live stream of container(s) resource usage statistics",
 		Args:  cli.RequiresMinArgs(0),
```

Comment:
@allencloud  PTAL
Agree with it.  I will apply it to this PR. 
Could you have a suggestion about the option '--format'  for docker stats?
Do docker stats support the option '--option' which statsOptions includes format in code?  
Thanks